helpers/handlebarsHelpers.js

```diff
@@ -17,56 +17,48 @@ module.exports = {
         );
   },
   addOpenGraph (title, mimeType, showUrl, source, description, thumbnail) {
-    let basicTags = `<meta property="og:title" content="${title}"> 
-          <meta property="og:url" content="${showUrl}" > 
-          <meta property="og:site_name" content="Spee.ch" > 
-          <meta property="og:description" content="${description}">`;
+    console.log('title', title);
+    console.log('description', description);
+    let ogTitle = `<meta property="og:title" content="${title}" >`;
+    let ogUrl = `<meta property="og:url" content="${showUrl}" >`;
+    let ogSiteName = `<meta property="og:site_name" content="Spee.ch" >`;
+    let ogDescription = `<meta property="og:description" content="${description}" >`;
```

Comment:
Is there a fallback value for this when it is not set?

Reply:
no there isn't, and you are correct that should be added :)